### PR TITLE
HOTFIX: Fix unsafe dependence on class name in VerifiableClientJava

### PR DESCRIPTION
**Pull request by hachikuji**

### CODE REVIEW

File: tests/kafkatest/services/verifiable_consumer.py [hunk 1]
```diff
@@ -129,7 +129,14 @@ def last_commit(self, tp):
             return None
 
 
-
+"""
```
Comment (ewencp) [inline tests/kafkatest/services/verifiable_consumer.py]: docstrings go below the class (unlike every other language). same with other changes in this patch

**Review by ewencp — COMMENTED** [review]
Wow, totally did not catch this during the review of the changes for pluggable clients.

One very minor cleanup issue, then LGTM

**Comment by hachikuji** [comment]
FWIW, here's the build off this branch which is looking good so far: https://jenkins.confluent.io/view/All/job/system-test-confluent-platform-branch-builder/195/console

**Comment by ewencp** [comment]
LGTM, no point waiting for jenkins since this is all system test code. Will merge to trunk.

**Pull request closed without merging**